swipe.js

```diff
@@ -0,0 +1,58 @@
+var body = document.body;
+var endpoint = window.location.href.split('/')[7];
+var forward, backward;
+
+// Homepage to choose an avatar page
+if (endpoint === 'index.html'){
```

Comment:
Something like this would be better
```js
getBackForward = pageName => {
  const dirs = {
    ['index.html']: ['choose-an-avatar.html', ''],
    ['choose-an-avatar.html']: ['introduction.html', 'index.html']
   ...
  }
  const dirsChosen = dirs[pageName];
  return {
    forward: dirsChosen[0],
    backward: dirsChosen[1];
  };

}